.travis.yml

```diff
@@ -1,8 +1,31 @@
-language: objective-c
-
 notifications:
   email:
     on_success: never
     on_failure: change
 
 script: 'curl -s https://raw.githubusercontent.com/atom/ci/master/build-package.sh | sh'
+
+git:
+  depth: 10
```

Comment:
It might make problems!

See: https://docs.travis-ci.com/user/customizing-the-build/#Git-Clone-Depth
> Please note that if you use a depth of 1 and have a queue of jobs, Travis CI won’t build commits that are in the queue when you push a new commit.